taxi/urls.py

```diff
@@ -1,9 +1,26 @@
 from django.urls import path
 
-from .views import index
+from .views import (
+    index,
+    ManufacturerListView,
+    CarListView,
+    CarDetailView,
+    DriverListView,
+    DriverDetailView,
+)
+
 
 urlpatterns = [
     path("", index, name="index"),
+    path(
+        "manufacturers/",
+        ManufacturerListView.as_view(),
+        name="manufacturer-list",
+    ),
+    path("cars/", CarListView.as_view(), name="car-list"),
+    path("drivers/", DriverListView.as_view(), name="driver-list"),
+    path("cars/<int:pk>", CarDetailView.as_view(), name="car-detail"),
```

Comment:
```suggestion
    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
```
don't forget about this